validator/sawtooth_validator/gossip/gossip.py

```diff
@@ -26,20 +28,37 @@ def broadcast_block(self, block):
             content_type="BLOCK",
             content=block.SerializeToString())
 
-        self._broadcast(gossip_message)
+        self._broadcast(gossip_message, validator_pb2.Message.GOSSIP_MESSAGE)
+
+    def broadcast_block_request(self, block_id):
+        # Need to define node identity to be able to route directly back
+        block_request = GossipBlockRequest(block_id=block_id)
+        self._broadcast(block_request,
+                        validator_pb2.Message.GOSSIP_BLOCK_REQUEST)
 
     def broadcast_batch(self, batch):
         gossip_message = GossipMessage(
             content_type="BATCH",
             content=batch.SerializeToString())
 
-        self._broadcast(gossip_message)
+        self._broadcast(gossip_message, validator_pb2.Message.GOSSIP_MESSAGE)
+
+    def broadcast_batch_request(self, batch_id, id_type):
+        # Need to define node identity to be able to route directly back
+        batch_request = GossipBatchRequest(
+            id_type=id_type,
+            id=batch_id
+        )
+        self._broadcast(batch_request,
+                        validator_pb2.Message.GOSSIP_BATCH_REQUEST)
 
-    def _broadcast(self, gossip_message):
-        message_type = validator_pb2.Message.GOSSIP_MESSAGE
+    def _broadcast(self, gossip_message, message_type):
         for connection in self._network.connections:
             connection.send(message_type, gossip_message.SerializeToString())
 
+    def broadcst(self, gossip_message, message_type):
+        self._broadcast(gossip_message, message_type)
```

Comment:
Why do we need a public method that just calls the private method? Can we just have a single public method?